depfinder/cli.py

```diff
@@ -120,8 +120,8 @@ def _init_parser():
     )
     p.add_argument(
         '--ignore',
-        default=None,
-        help="Blacklist pattern for files not to inpsect"
+        default='',
+        help="Ignore pattern(s) for files not to inpsect"
```

Comment:
Can you add that this can take a comma separated list of strings for multiple patterns?